api/server/router/network/network_routes.go

```diff
@@ -100,6 +100,24 @@ func (e ambigousResultsError) Error() string {
 
 func (ambigousResultsError) InvalidParameter() {}
 
+type conflictingNetworkName struct {
```

Comment:
This would probably be best as just a generic conflict error

Reply:
Yes, you're right! A generic conflict error will be good to extend.